servers/zms/src/test/resources/schema/zms_server.sql

```diff
@@ -0,0 +1,357 @@
+-- MySQL Script generated by MySQL Workbench
+-- Tue Jun 30 12:33:10 2020
+-- Model: New Model    Version: 1.0
+-- MySQL Workbench Forward Engineering
```

Comment:
Is this file different from the original zms_server.sql? If it isn't that means we will need to remember updating this file whenever we update the original.
Can we reference the original file instead of copying it to the test directory?

Reply:
It's the same file for now. remembering part is not a concern since if you write unit tests and forget to update the sql file, your tests are going to fail anyway. but the library is based around resources and I have to review their actual code to see how I can specify a relative path (which might be different depending what. class we're calling this from??).